docs/cellxgene_census_docsite_data_release_info.md

```diff
@@ -1,23 +1,110 @@
 # Census data releases 
 
-**Last edited**: April, 2023.
+**Last edited**: July, 2023.
+
+**Contents**
+
+1. [What is a Census data release?](#What-is-a-Census-data-release)
+2. [List of LTS Census data releases](#List-of-LTS-Census-data-releases)
 
 ## What is a Census data release?
 
 It is a Census build that is publicly hosted online. A Census build is 
 a [TileDB-SOMA](https://github.com/single-cell-data/TileDB-SOMA) collection with the Census data from [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) as specified in the [Census schema](cellxgene_census_docsite_schema.md). 
 
-### Long-term supported (LTS) Cell Census releases
+### Long-term supported (LTS) Census releases
 
 To enable data stability and scientific reproducibility, [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) plans to perform regular LTS Census data releases:
 
 * Published online every six months for public access, starting on May 15, 2023.
 * Available for public access for at least 5 years upon publication.
  
+The latest LTS Census data release is the default opened by the APIs and recognized as the `census_version = "stable"`.
+
+Python
+
+```python
+import cellxgene_census
```

Comment:
did you want to have these code samples explicitly specify `census_version="stable"`, rather than relying on the default (which could, in theory, change)